src/authentication/middleware/index.ts

```diff
@@ -7,7 +7,7 @@ export default (req: Request, res: Response, next: NextFunction) => {
   const referringPageURL = req.header("Referer") as string;
 
   logger.debug("Check if user has referer");
-  if (referringPageURL === undefined) {
+  if (referringPageURL === undefined && !req.originalUrl.endsWith("/download")) {
```

Comment:
Could you make use of the newly defined DOWNLOAD_PREFIX constant you defined in page.urls.ts here?

Reply:
👍